### PR TITLE
Generate ext/intl class synopses from stubs - part 2

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/intl/intlbreakiterator.xml [hunk 1]
```diff
@@ -38,19 +37,20 @@
...
 
-<!-- {{{ Class synopsis -->
     <classsynopsisinfo>
      <ooclass>
       <classname>IntlBreakIterator</classname>
      </ooclass>
 
      <oointerface>
-      <interfacename>Traversable</interfacename>
+      <interfacename>IteratorAggregate</interfacename>
```
Comment (cmb69) [inline reference/intl/intlbreakiterator.xml:50]: This is only as of PHP 8.0.0, so needs a changelog entry. Also, the introduction section above explicitly mentions Traversable, what should probably be changed right away.

**Review by cmb69 — COMMENTED** [review]
Thank you! Looks good besides the IntlBreakIterator issues mentioned below.

File: reference/intl/intlbreakiterator.xml (hunk 1 above)
Comment (kocsismate) [reply reference/intl/intlbreakiterator.xml:50]: Ah, yes, I didn't notice these ones!

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**